### PR TITLE
Update existing constant aquifer flux test model

### PR DESCRIPTION
**Pull request by tskille**
This is a better test case for constant aquifer flux. This is copied of AQUNUM-01.DATA where the numerical aquifer is replaced with a constant flux aquifer.

### CODE REVIEW

**Comment by tskille** [comment]
This case is yet not under regression testing, hence an jemkins build is not required.

File: aquifers/AQUFLUX-01.DATA [hunk 1]
```diff
@@ -19,289 +18,273 @@ RUNSPEC
...
- 
+ 1 /
 AAQR
- 1 2 /
- 
+ 1 /
 AAQT
- 1 2 /
-
+ 1 /
```
Comment (bska) [inline aquifers/AQUFLUX-01.DATA:230]: The header says "numerical aquifer", but the section requests `AAQ*` summary vectors.  At least one of the two is questionable to me.  Given that `AQUFLUX` uses analytic aquifer connections I would think that `AAQ*` would be the appropriate in which case the headers should probably refer to "analytic aquifers" instead.

**Review by bska — COMMENTED** [review]
This looks good for the most part, except for a slightly confusing set of summary vectors for the aquifer.

File: aquifers/AQUFLUX-01.DATA (hunk 1 above)
Comment (tskille) [reply aquifers/AQUFLUX-01.DATA]: Thanks @bska , my bad. I started out by copying an numerical aquifer case and forgot to change this. Updated now

**Review by bska — APPROVED** [review]
Thanks for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**